custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -655,15 +655,15 @@ def test_demographics_export_with_beta(self):
                             'Number of out of school adolescent girls (11-14 years)',
                             'Number of adolescent girls 15 to 18 years old', 'Total number of households members'],
                            ['st1', 2637, 369, 1518, '24.31 %', 120, 120, 171, 171, 1227, 1227, 56, 244, 927, 26, 0,
-                            12, 859],
+                            10, 859],
```

Comment:
Why is it a decrease of 2 and everywhere else a decrease of 1?

Reply:
as there is no month filter. We have 6 results for '2017-04-01' and 4 for '2017-05-01'

Reply:
So you mean the case you migrated was part of 2017-04-01 and 2017-05-01 both and therefore it reduced by 2?

Reply:
that is correct